lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -93,6 +93,11 @@
         - Client or application ID for azure_rm type.
       required: False
       default: null
+    security_token:
```

Comment:
@AlanCoding while writing these tests, I found that AWS STS Token just isn't supported at all.